frontend/src/app/main/data/workspace/libraries.cljs

```diff
@@ -98,19 +99,13 @@
       item
       (assoc  item :path path :name name))))
 
-(defn default-color-name [color]
-  (or (:color color)
-      (case (get-in color [:gradient :type])
-        :linear (tr "workspace.gradients.linear")
-        :radial (tr "workspace.gradients.radial"))))
-
 (defn add-color
   [color]
   (let [id    (uuid/next)
         color (-> color
                   (assoc :id id)
-                  (assoc :name (default-color-name color)))]
-    (dm/assert! (ctc/color? color))
+                  (assoc :name (uc/gradient-type->string (get-in color [:gradient :type]))))]
```

Comment:
This gets an empty name if there is no gradient, only a plain color. In this case the name should be the #rgb